pkg/sriov/sriov.go

```diff
@@ -374,6 +364,22 @@ func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 	return nil
 }
 
+// Fills original VF state
+func (s *sriovManager) FillOriginalVfInfo(conf *sriovtypes.NetConf) (netlink.Link, error) {
```

Comment:
can you add unit tests for this new method ?

Reply:
I have made the changes that you have asked, kindly check it.